src/main/java/com/TheAccountant/controller/LoanController.java

```diff
@@ -0,0 +1,83 @@
+package com.TheAccountant.controller;
+
+import com.TheAccountant.controller.exception.ConflictException;
+import com.TheAccountant.controller.exception.NotFoundException;
+import com.TheAccountant.dao.LoanDao;
+import com.TheAccountant.model.loan.Loan;
+import com.TheAccountant.model.user.AppUser;
+import com.TheAccountant.util.UserUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.transaction.Transactional;
+import javax.validation.Valid;
+import java.util.List;
+
+/**
+ * Created by tudor.grigoriu on 3/17/2017.
+ */
+@RestController
+@RequestMapping(value = "/loans")
+public class LoanController {
+
+    @Autowired
+    LoanDao loanDao;
+
+    @Autowired
+    private UserUtil userUtil;
+
+    @RequestMapping(method = RequestMethod.POST)
+    @Transactional
+    public ResponseEntity<Loan> create(@RequestBody @Valid Loan loan) {
+        try {
+            Loan createdLoan = loanDao.saveAndFlush(loan);
```

Comment:
Should attach the logged AppUser to the Loan Entity, and also check if Counterparty is already attached to the entity